libs/filamentjs/docs/build.py

```diff
@@ -204,24 +204,32 @@ def spawn_local_server():
     import socketserver
     Handler = http.server.SimpleHTTPRequestHandler
     Handler.extensions_map.update({ '.wasm': 'application/wasm' })
-    Handler.directory = SERVER_DIR
-    os.chdir(SERVER_DIR)
+    server_dir = OUTPUT_DIR + '..'
```

Comment:
Maybe use `os.path.join`? Might avoid a bug where the user doesn't add a trailing slash to the `output_folder ` path

Reply:
Nice catch, fixed by simply removing the server_dir variable since there's no reason to serve the parent folder.